specifyweb/frontend/js_src/lib/components/WorkBench/hotProps.tsx

```diff
@@ -62,23 +61,20 @@ export function useHotProps({
 
   const colHeaders = React.useCallback(
     (physicalCol: number) => {
-      const tableIcon = mappings?.mappedHeaders?.[physicalCol];
-      const isMapped = tableIcon !== undefined;
+      const tableIconUrl = mappings?.mappedHeaders?.[physicalCol];
+      const isMapped = tableIconUrl !== undefined;
       const mappingCol = physicalColToMappingCol(physicalCol);
       const tableName =
         (typeof mappingCol === 'number'
           ? mappings?.tableNames[mappingCol]
-          : undefined) ?? tableIcon?.split('/').slice(-1)?.[0]?.split('.')?.[0];
-      const tableLabel = isMapped
-        ? f.maybe(tableName, getTable)?.label ?? tableName ?? ''
-        : '';
-      // REFACTOR: use new table icons
+          : undefined) ??
+        tableIconUrl?.split('/').slice(-1)?.[0]?.split('.')?.[0];
```

Comment:
.slice() never returns undefned - no need for `?.`

```suggestion
        tableIconUrl?.split('/').slice(-1)[0]?.split('.')[0];
```